src/commands/BootstrapCommand.js

```diff
@@ -28,8 +28,10 @@ export default class BootstrapCommand extends Command {
    * @param {Function} callback
    */
   bootstrapPackages(callback) {
-    this.filteredGraph = PackageUtilities.getPackageGraph(this.filteredPackages);
     this.logger.info(`Bootstrapping ${this.filteredPackages.length} packages`);
+    this.batchedPackages = this.flags.toposort
```

Comment:
The most foolproof correct way to implement this flag would then be to default it on always, and provide a `--no-sort` option to disable it, no? (Agree that "topo" is probably noisy; I think "no sort" is phrased the most CLI-like though.)